R/reportSE.R

```diff
@@ -372,8 +372,10 @@ reportSE <- function(gdx, regionSubsetList = NULL, t = c(seq(2005, 2060, 5), seq
   vm_demFeSector[is.na(vm_demFeSector)] <- 0
   # SE demand
   vm_demSe <- readGDX(gdx, "vm_demSe", field = "l", restore_zeros = F)[, y, ] * pm_conv_TWa_EJ
-  # SE demand of specific energy system technologies
-  v_demSeOth <- readGDX(gdx, c("v_demSeOth","vm_demSeOth"), field = "l", restore_zeros = T)[, y, ] * pm_conv_TWa_EJ
+  # SE demand of specific energy system technologies (ensure that all regions have a value)
+  v_demSeOth <- readGDX(gdx, c("v_demSeOth", "vm_demSeOth"), field = "l", restore_zeros = FALSE)[, y, ] * pm_conv_TWa_EJ
+  matrixRegionsYears <- new.magpie(cells_and_regions = getRegions(dataoc), years = y, fill = 0, sets = getSets(v_demSeOth, fulldim = FALSE))
```

Comment:
Not sure if it is necessary to create a dummy magclass object just for the matching here. Usually, the reference `ref` given to `matchDim` is the magclass object you which causes `mbind` to crash.

Reply:
Hey,
I tried to matchDim v_demSeOth with tmp1, but the dimension names don't match. That's why i used an intermediate named object.
What would be a more fitting solution?

Reply:
Falk's solution here: https://github.com/pik-piam/remind2/pull/644